fdbclient/include/fdbclient/StorageServerInterface.h

```diff
@@ -1266,6 +1270,51 @@ struct GetHotShardsRequest {
 	}
 };
 
+struct CheckSumMetaData {
+	constexpr static FileIdentifier file_identifier = 3828142;
+	KeyRange range;
+	Version version;
+	StringRef checkSumValue;
```

Comment:
Can you add another field for checksum/hash algorithm? Ideally, this gives us the freedom to implement different algorithms.